examples/demo.css

```diff
@@ -153,25 +153,19 @@ h1 {
   height: auto;
 }
 
-.move-up {
+#move-up {
     position: fixed;
     bottom: 50px;
     right: 40px;
-    z-index: 3;
+    z-index: 99;
```

Comment:
Some classes in bootstrap have a z-indez of even `1000` so I think `550` would be better here.